src/lib/response-body-listener.ts

```diff
@@ -20,17 +20,22 @@ export class ResponseBodyListener {
       details.requestId,
     ) as any;
 
-    const decoder = new TextDecoder("utf-8");
+    // const decoder = new TextDecoder("utf-8");
```

Comment:
Please remove the commented out code. We're planning a broader cleanup of all the commented code in this repo, but we may as well do it here since we're updating this file.

Reply:
Has been done in ca7e2c714cdb5eb4a7758e0cac4fa690a90f0154